src/Mod/Part/App/BezierCurvePy.xml

```diff
@@ -148,17 +148,17 @@ If this Bezier curve is rational, it can become non-rational.</UserDocu>
 for a given 3D tolerance (Tolerance3D).
 If f(t) is the equation of this Bezier curve, the parametric tolerance
 ensures that:
-|t1-t0| &lt; UTolerance =&quot;&quot;==&gt; |f(t1)-f(t0)| &lt; Tolerance3D</UserDocu>
+|t1-t0| &lt; UTolerance ==&gt; |f(t1)-f(t0)| &lt; Tolerance3D</UserDocu>
```

Comment:
Looks to me like someone wrote `=“”==>` by choice, should that really be removed?

Reply:
👍 (resolved)